src/query-builder/expression-builder.ts

```diff
@@ -58,8 +58,8 @@ export class ExpressionBuilder<DB, TB extends keyof DB> {
    * having count("pet"."id") > $1
    * ```
    */
-  get fn(): FunctionBuilder<DB, TB> {
-    return new FunctionBuilder()
+  get fn(): FunctionModule<DB, TB> {
+    return new FunctionModule()
```

Comment:
Nice change!

Reply:
Thanks! Was a bit scared of this, since this is a breaking change (if anything ever used the type directly).